Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -371,19 +374,23 @@ public final class PackageInfoMapper: PackageInfoMapping {
             return nil
         }
 
+        let options: ProjectDescription.Project.Options
+        if let configuration = configuration {
+            let config = ProjectDescription.Project.Configuration.from(manifest: configuration)
+            options = config.options
+        } else {
+            options = .options(automaticSchemesOptions: .disabled)
```

Comment:
Before we were passing `disableSynthesizedResourceAccessors: false` but `resourceSynthesizers: []`, to keep the same default behavior we need to set `disableSynthesizedResourceAccessors: true` given we are now passing `resourceSynthesizers: .default`
```suggestion
            options = .options(
              automaticSchemesOptions: .disabled,
              disableSynthesizedResourceAccessors: true
            )
```